centreon/src/Core/Dashboard/Infrastructure/API/AddDashboard/AddDashboardPresenter.php

```diff
@@ -88,11 +88,11 @@ public function presentResponse(AddDashboardResponse|ResponseStatusInterface $da
     }
 
     /**
-     * @param ?AddDashboardUserDto $dto
+     * @param ?\Core\Dashboard\Application\UseCase\AddDashboard\Response\UserResponseDto $dto
```

Comment:
```suggestion
     * @param ?UserResponseDto $dto
```

Namespace is already in the uses